app/templates/admin/manage_user.html

```diff
@@ -32,9 +32,13 @@
 {% block content %}
     <div class="ui stackable centered grid container">
         <div class="twelve wide column">
-            <a class="ui basic compact button" href="{{ url_for('admin.registered_users') }}">
+            <a class="ui basic compact button" href="{{ url_for('admin.approved_users') }}">
                 <i class="caret left icon"></i>
-                Back to all users
+                Back to all approved users
```

Comment:
if the user is unapproved, make the button go back to all unapproved users, if it is approved, have it go back to all approved users -- the two buttons are confusing